cli/src/semgrep/app/scans.py

```diff
@@ -266,9 +266,11 @@ def report_findings(
         contributions: out.Contributions,
         engine_requested: "EngineType",
         progress_bar: "Progress",
-    ) -> Tuple[bool, str]:
+    ) -> Tuple[bool, bool, str]:
         """
         commit_date here for legacy reasons. epoch time of latest commit
+
+        Returns (success, block_scan, block_reason)
```

Comment:
At this point you can define a @dataclass ScanResult with actual field names instead of those Tuple that require comments.